source/index.md

```diff
@@ -17,17 +17,19 @@ Meteor is a full-stack JavaScript platform for developing modern web and mobile
 
 <h2 id="learning-more">Meteor resources</h2>
 
-1. The place to get started with Meteor is the [tutorials page](https://www.meteor.com/developers/tutorials).
+1. First, learn how to install meteor in the [installation section](/install.html).
 
-1. [Meteor Examples](https://github.com/meteor/examples) is a list of examples using Meteor. You can also include your example with Meteor.
+2. The place to get started with Meteor is the [tutorials page](https://www.meteor.com/developers/tutorials).
```

Comment:
Should be all 1 so we don't need to worry about numbers. This is a markdown feature.

Reply:
webstorm changed everything :facepalm: